CHANGELOG.unreleased.md

```diff
@@ -20,6 +20,8 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [REST API] Correctly return a 404 not found error when trying to get a backup log that does not exist (PR [#8457](https://github.com/vatesfr/xen-orchestra/pull/8457))
+- **XO 6:**
+  - [Host] Fix RAM usage progress bar broken when memory used and total units were different PR [#8461](https://github.com/vatesfr/xen-orchestra/pull/8461)
```

Comment:
There is no need to add a changelog, as the host dashboard is not yet released. (So only source users have noticed the bug)
```suggestion
```